napalm_ios/ios.py

```diff
@@ -20,6 +20,8 @@
 import os
 import uuid
 import tempfile
+import copy
+import ipaddress
```

Comment:
That function provides no way (that I can see) to insist on a v4 vs. v6 address (which we need for remote_id).
Since `ipaddress` is now in the stdlib from `py34`, is there a reason to prefer the use of `netaddr`? A performance difference that you're aware of maybe?